client/my-sites/activity/activity-log-item/activity-actor.jsx

```diff
@@ -42,6 +44,15 @@ const WORDPRESS_ACTOR = (
 	</div>
 );
 
+const MULTIPLE_ACTORS = (
+	<div className="activity-log-item__actor">
+		<ActivityActorIcon icon="multiple-users" />
+		<div className="activity-log-item__actor-info">
+			<div className="activity-log-item__actor-name">{ translate( 'Multiple Users' ) }</div>
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Multiple users' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).